mne/tests/test_epochs.py

```diff
@@ -57,6 +59,48 @@
 rng = np.random.RandomState(42)
 
 
+def test_handle_duplicate_events():
+    """Test handling of duplicate events."""
+    # A general test case
+    EVENT_ID = {'aud': 1, 'vis': 2}
+    EVENTS = np.array([[0, 0, 1], [0, 0, 2]])
```

Comment:
why capital letters?

Reply:
because these are my "constants" for the test

Reply:
I would rename them `EXPECTED_BLA` and I would not use them on the input. ;)